gbpservice/neutron/nsf/configurator/drivers/vpn/vyos/vyos_vpn_driver.py

```diff
@@ -0,0 +1,642 @@
+# One Convergence, Inc. CONFIDENTIAL
+# Copyright (c) 2012-2016, One Convergence, Inc., USA
+# All Rights Reserved.
+#
+# All information contained herein is, and remains the property of
+# One Convergence, Inc. and its suppliers, if any. The intellectual and
+# technical concepts contained herein are proprietary to One Convergence,
+# Inc. and its suppliers.
+#
+# Dissemination of this information or reproduction of this material is
+# strictly forbidden unless prior written permission is obtained from
+# One Convergence, Inc., USA
+
+import copy
+import json
+import re
+import requests
+
+from oslo_concurrency import lockutils
+from oslo_config import cfg
+from oslo_log import log as logging
+from gbpservice.neutron.nsf.configurator.drivers.base.\
+                            base_driver import BaseDriver
+from gbpservice.neutron.nsf.configurator.lib import exceptions as exc
+from gbpservice.neutron.nsf.configurator.lib import vpn_constants as const
+from gbpservice.neutron.nsf.configurator.agents.vpn import RestApi
+from gbpservice.neutron.nsf.configurator.agents.vpn import VPNSvcValidator
+
+LOG = logging.getLogger(__name__)
+
+
+class VpnGenericConfigDriver(object):
+    """
+    Driver class for implementing VPN configuration
+    requests from Orchestrator.
+    """
+
+    def __init__(self):
+        self.timeout = cfg.CONF.rest_timeout
+
+    def configure_source_routes(self, context, kwargs):
```

Comment:
Rename the function to configure_routes and clear_routes instead of source_routes.